src/wam2layers/utils/profiling.py

```diff
@@ -56,8 +58,7 @@ def store_intermediate_states(self, output):
             self.tracked += totals["p_track_lower"] + totals["p_track_upper"]
             self.total_tagged_moisture += totals["tagged_evap"]
 
-        # Don't include boundary losses in log messages
-        interior_losses = output.losses[1:-1, 1:-1].sum()
+        interior_losses = area_weighted_output.losses[1:-1, 1:-1].sum()
```

Comment:
For periodic boundaries this is not completely correct. Maybe not solve it now, but just add a 'TODO' or note about it at least?